docs/details/dashboards/dashboard-inventory.md

```diff
@@ -8,11 +8,23 @@ To check your inventory list, go to <i class="uil uil-cog"></i> **Configuration*
 
 Inventory objects form a hierarchy with Node at the top, then Service and Agents assigned to a Node. This information is detailed in the two tabs available on this page.
 
-### **Services** tab
+### Services tab
+
+The **Services** tab displays the individual services, the nodes on which they run, and the Agents that help collect the service metrics along with the following information:
+
+**Status** - The **Status** column shows the status of your databases based on metrics coming directly from the database.
```

Comment:
I would remove `The Status column` as it's probably redundant.

Reply:
Done.